char defaults to unsigned on ARM, causing a test failure.
Patch from upstream.
https://bugs.gentoo.org/811927
https://sourceforge.net/p/gnuplot/bugs/2467/

--- a/src/marching_cubes.h
+++ b/src/marching_cubes.h
@@ -58,7 +58,7 @@ static const short cube_edge_flags[256]=
          First official public domain release
               Version 3.00 dated 8-6-94
  */
-static char triangle_table[256][13]=
+static signed char triangle_table[256][13]=
 {
     {-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1},
     { 8, 3, 0,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1},
--- a/src/qt_table.h
+++ b/src/qt_table.h
@@ -10,7 +10,7 @@
  *     collapsed them into quadrangles.
  *     So there is room for improvement in this table.
  */ 
-static char qt_table[256][13] = 
+static signed char qt_table[256][13] = 
 {
     {  -1, -1, -1, -1,  -1, -1, -1, -1,  -1, -1, -1, -1,  -1},
     {   8,  3,  0, -1,  -1, -1, -1, -1,  -1, -1, -1, -1,  -1},
